enhancements/rhcos/liveisoinstall.md

```diff
@@ -0,0 +1,93 @@
+---
+title: RHEL CoreOS Live ISO/installer
+authors:
+  - "@cgwalters"
+reviewers:
+  - "@coreos-team"
+approvers:
+  - "@coreos-team"
+creation-date: 2020-02-07
+last-updated: 2020-02-07
+status: implementable
+---
+
+# Rebase RHEL CoreOS 4.5 on Fedora CoreOS $latest
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+RHEL CoreOS is built from RHEL components, but uses configuration and tooling from Fedora CoreOS.  They share many things such as the installer code, etc.
+
+This enhancement tracks rebasing RHCOS 4.5 on the latest FCOS code, in particular the installer.
+
+### New "Live" installer
+
+Currently, RHCOS 4.4 and below offer a kernel/initramfs and ISO that can run a shell script installation program from the initramfs.
+
+In FCOS $latest, there is a Live ISO/PXE media that runs the full OS (for example, it includes `podman`), and a [coreos-installer](https://github.com/coreos/coreos-installer/) program was rewritten.
+
+## Motivation
+
+This allows administrators to run arbitrary code before and after the installation process by providing Ignition that wraps the installer.  For example, higher level tooling like Foreman and Ironic can easily `curl` a URL to report provisioning success, or even pull down and launch a container before/after.
+
+An additional goal is to allow administrators to more easily run RHCOS as a "live" system for *interactive* experiementation and testing.  For example, an administrator could run the system from a USB stick to validate the network interface names, etc.
+
+### Goals
+
+A live ISO and new `coreos-installer` binary are shipped, tested and work in all scenarios supported by the existing installer, and contain the new functionality.
+
+### Non-Goals
+
+Support for Live/diskless nodes joining a cluster.
+
+## Proposal
+
+Switch the RHCOS build process to generate the new Live ISO media.
+
+Add a basic (virtualized) test case for the installer.
+
+Switch the `e2e-metal` job in Packet to use the new installer.
```

Comment:
hmm, we use iPXE for metal on Packet.net. How will this help the e2e-metal ?

Reply:
I don't think it will *help* it offhand, things should work the same. Although, Packet AFAIK has a "checkin" procedure that we could be using perhaps and aren't?

This is just about extending capabilities before/after install; may not help specific cases but shouldn't break them either.